### PR TITLE
add updated content for new demo release

### PR DESCRIPTION
**Pull request by austinlparker**
This is waiting on https://github.com/open-telemetry/opentelemetry-demo/pull/1211

---

**Preview**:

- https://deploy-preview-3486--opentelemetry.netlify.app/blog/2023/otel-demo-16/
- https://deploy-preview-3486--opentelemetry.netlify.app/docs/demo/features/
- https://deploy-preview-3486--opentelemetry.netlify.app/docs/demo/logging-features/

### CODE REVIEW

File: content/en/blog/2023/otel-demo-16.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: OpenTelemetry Demo 1.6 released
+linkTitle: Demo 1.6 Released
+date: 2023-11-03
+author: '[Austin Parker](https://github.com/austinlparker)'
+---
+
+The OpenTelemetry Demo has been updated to version 1.6, and introduces alpha
+support for the OpenTelemetry Log signal!
```
Comment (chalin) [inline content/en/blog/2023/otel-demo-16.md:9]:
```suggestion
The OpenTelemetry Demo has been updated to version 1.6, and introduces alpha
support for the OpenTelemetry [Log signal](/docs/concepts/signals/logs/)!
```

File: content/en/blog/2023/otel-demo-16.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Our thoughts on logs
+
+Why are we considering logging support to be in alpha? There's a few reasons.
```
Comment (chalin) [inline content/en/blog/2023/otel-demo-16.md:13]:
```suggestion
Why are we considering logging support to be in alpha? There are a few reasons.
```

File: content/en/blog/2023/otel-demo-16.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+The most important is that this is the first time we've shipped the demo with a
+logging database. We chose [OpenSearch](https://opensearch.org/) as the storage
+backend for logs because it's a popular option that aligns with the project's
+license. It's worthwhile to note, though, that the demo has been designed to not
+really need application-level logs -- everything you care about is available as
+a span.
```
Comment (chalin) [inline content/en/blog/2023/otel-demo-16.md:19]:
```suggestion
license. It's worth noting that the demo doesn't require application-level logs &mdash; everything you care about is available as a span.
```

File: content/en/blog/2023/otel-demo-16.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Where logs shine is in bridging the gap between application traces and
+infrastructure, and over the next few releases we plan to enhance this story by
```
Comment (chalin) [inline content/en/blog/2023/otel-demo-16.md:22]:
```suggestion
infrastructure, and over the next few releases, we plan to enhance this story by
```

File: content/en/blog/2023/otel-demo-16.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+collecting Kubernetes, Envoy, and other logs that are relevant to the demo.
```
Comment (chalin) [inline content/en/blog/2023/otel-demo-16.md:23]:
```suggestion
collecting Kubernetes, Envoy, and other logs relevant to the demo.
```

File: content/en/blog/2023/otel-demo-16.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+If this sounds like an interesting problem to work on, we'd love to have you on
+board as a
```
Comment (chalin) [inline content/en/blog/2023/otel-demo-16.md:26]:
```suggestion
If this sounds like an interesting problem, we'd love to have you as a
```

File: content/en/blog/2023/otel-demo-16.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+be able to increase the number of replicas per service, for instance. We've
+updated many dependencies throughout the project as well.
+
+## What's next?
+
+With the addition of logging, the demo is mostly feature-complete. However,
+there's more that can be done, and we're looking at a few major areas over the
+next months:
+
+- Increase the number of documented scenarios, to demonstrate how OpenTelemetry
```
Comment (chalin) [inline content/en/blog/2023/otel-demo-16.md:43]:
```suggestion
- Increase the number of documented scenarios to demonstrate how OpenTelemetry
```

File: content/en/blog/2023/otel-demo-16.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+  can be used to solve real-world problems.
+- Ensure that the demo reflects the OpenTelemetry feature matrix, and that we're
+  showing off everything you can do with the project.
```
Comment (chalin) [inline content/en/blog/2023/otel-demo-16.md:46]:
```suggestion
- Ensure that the demo reflects the OpenTelemetry feature matrix and that we're
  showing off everything you can do with the project.
```

File: content/en/blog/2023/otel-demo-16.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+- Improve our dashboards, add alerts, SLOs, and other observability features to
```
Comment (chalin) [inline content/en/blog/2023/otel-demo-16.md:47]:
```suggestion
- Improve our dashboards and add alerts, SLOs, and other observability features to
```

**Review by chalin — APPROVED** [review]
Looks good. A few copyedits.

You might want to search-and-replace: `s/ -- / &mdash /g`.

**Review by mviitane — APPROVED** [review]

**Comment by cartermp** [comment]
Looks like action won't kick in :/

**Comment by chalin** [comment]
Yeah, I'm suspecting that it's a permissions issue with the GH action. What we need is a real OTel bot with all the necessary permissions. Maybe @trask can offer guidance :).

**Review by cartermp — APPROVED** [review]

**Comment by cartermp** [comment]
Merging when green, as the upstream has already been released

**Pull request merged**